src/components/ClosedQueueCard.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardBody, CardTitle, CardSubtitle, Button } from 'reactstrap'
+import FontAwesomeIcon from '@fortawesome/react-fontawesome'
+import faMapMarker from '@fortawesome/fontawesome-free-solid/faMapMarker'
+import faQuestion from '@fortawesome/fontawesome-free-solid/faQuestionCircle'
+
+import ShowForCourseStaff from './ShowForCourseStaff'
+
+const ClosedQueueCard = ({
```

Comment:
This seems like a lot of code duplication - might be better to just have `QueueCard` take an `open` prop?

Reply:
I was trying this way earlier, I can do it like this, but if I did, then closed cards would have the hover animation. If thats okay, then I can switch it so they use the same component

Reply:
earlier an issue was with the color of the card too, but now that would be an easy fix, so hover is the only issue

Reply:
You should be able to fix that by just applying a different set of styles?

Reply:
v true, i was trying something overly complicated last night lol